apps/web/src/pages/Profile/ExperienceFormPage/ExperienceForm.test.tsx

```diff
@@ -343,22 +343,21 @@ describe("ExperienceForm", () => {
     await waitFor(() => expect(screen.queryAllByRole("alert")).toHaveLength(0));
   });
 
-  // TODO: Commenting out test below until the <SkillBrowserDialog /> error is resolved... When skill dialog is opened this console.error() appears -> "Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?"
-  // it("should add skill", async () => {
-  //   renderExperienceForm({
-  //     userId: mockUserId,
-  //     experienceType: "award",
-  //     skillsQuery: skillFragments,
-  //   });
-
-  //   await act(() => {
-  //     screen
-  //       .getAllByRole("button", {
-  //         name: /add a skill/i,
-  //       })[0]
-  //       .click();
-  //   });
-  // });
+  it("should add skill", async () => {
+    renderExperienceForm({
+      userId: mockUserId,
+      experienceType: "award",
+      skillsQuery: skillFragments,
+    });
+
+    await act(() => {
```

Comment:
We can drop the await here

Reply:
d57c35d2987d192eef07aaaabddaa15f375a32c8